package.json

```diff
@@ -37,5 +38,8 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "devDependencies": {
+    "firebase-tools": "^10.6.0"
```

Comment:
WHy have this in your dependencies, shouldnt this be installed locally than in the repo deps?